src/pages/VolunteersFind.page

```diff
@@ -41,16 +41,8 @@
     </style>
     <script type="text/javascript">
     (function($) {
-        //impossible to SLDS style visualforce multiselect picklists, instead we'll use a standard select
-        function fixMultiPicklists() {
-            $('.multiSelectPicklistTable').hide();
-            $('.multipicklistSLDSFixup').attr("size", 4);
```

Comment:
Sorry, I forgot to pull down your changes and was working off my old version, I'll put that back.